nucypher/utilities/prometheus/collector.py

```diff
@@ -432,65 +432,3 @@ def _event_occurred(self, event) -> None:
         super()._event_occurred(event)
         self.metrics["current_worker_is_me_gauge"].set(
             self.contract_agent.get_worker_from_staker(self.staker_address) == self.worker_address)
-
-
-class BidRefundCompositeEventMetricsCollector(MetricsCollector):
-    """
-    Collector for both Bid and Refund WorkLock events.
-
-    Both Bid and Refund events additionally update the same metric of how much eth is deposited by the staker
```

Comment:
I agree Bid is useless now, but Refund will be useful for a while

Reply:
✔️